README.md

```diff
@@ -166,7 +166,34 @@ return this.infinityModel("product", { perPage: 12, startingPage: 1,
                                        category: "furniture" });
 ```
 
-Bound Parameters has been DEPRECATED as the third argument to `infinityModel`.  
+**Extending infinityModel**
+
+You can customize your infinityModel through configuring it like so and pass as the third argument to infinityModel:
+
+```js
+const ExtendedInfinityModel = InfinityModel.extend({
```

Comment:
Can we please provide the import statement above here so folks don't have to read the source code?